crds/exoscale.appcat.vshn.io_exoscalekafkas.yaml

```diff
@@ -87,10 +87,11 @@ spec:
                           type: object
                           x-kubernetes-preserve-unknown-fields: true
                         version:
-                          default: "3.2"
-                          description: Version contains the minor version for Kafka. Currently only "3.2" is supported. Leave it empty to always get the latest supported version.
+                          default: "3.4"
+                          description: Version contains the minor version for Kafka. Currently only "3.4" is supported. Leave it empty to always get the latest supported version.
                           enum:
-                            - "3.2"
+                            - "3.3"
```

Comment:
there are 3.3 and 3.4, but in description is message, that only 3.4 is supported - does it make any sense to expose 3.3 anyway?

Reply:
Yeah let's remove 3.3